CHANGELOG.md

```diff
@@ -21,6 +21,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 - [Validation] `full_validation()` now checks whether a Dataset is IATI XML. [#239]
 - [Validation] Test that SSOT organisation test files are valid IATI XML. [#242]
 
+- [Versions] Add a class to represent Standard Versions. Handles current and proposed formats. [#273]
```

Comment:
I'd propose leaving this out, since if you're detailing SemVer then it also stands that current formats (IATIVer, Decimal) should also be enumerated, and that feels a bit messy since `IATIVer` isn't a commonly understood term.